cadquery/cq.py

```diff
@@ -698,7 +698,7 @@ def first(self: T) -> T:
     def item(self: T, i: int) -> T:
         """
 
-        Return the ith item on the stack.
+        Return the nth item on the stack.
```

Comment:
I think `ith` was correct here since the index variable was `i`.

Reply:
Sorry about that, reverted in a7f62f2